2010-11-04  Michael Tautschnig  <mt@debian.org>

	* setup-storage/Volumes.pm: Included parted output line if parsing parted
		output fails (the mysterious Table header not seen yet).
Index: trunk/lib/setup-storage/Volumes.pm
===================================================================
--- trunk.orig/lib/setup-storage/Volumes.pm
+++ trunk/lib/setup-storage/Volumes.pm	
@@ -251,7 +251,7 @@
         # we must have seen the header, otherwise probably the format has
         # changed
         defined ($cols{"File system"}{"start"})
-          or &FAI::internal_error("Table header not seen yet");
+          or &FAI::internal_error("Table header not yet seen while reading $line");
 
         # the info for the partition number
         my $num_cols_before = $cols{"Number"}{"start"};
